src/StackExchange.Redis/ConfigurationOptions.cs

```diff
@@ -904,19 +904,25 @@ private ConfigurationOptions DoParse(string configuration, bool ignoreUnknown)
                             {
                                 Tunnel = null;
                             }
-                            else if (value.StartsWith("http:"))
+                            else
                             {
-                                value = value.Substring(5);
-                                if (!Format.TryParseEndPoint(value, out var ep))
+                                // For backwards compatibility with `http:address_with_port`.
```

Comment:
IMO we should add tests for this change, i.e. that we get the expected values back out - even if that means poking at internal state after the parse

Reply:
I added the new format to the roundtrip test. This should be sufficient to confirm that the resulting `Tunnel` is the same for both input formats (`HttpProxyTunnel.ToString()` internally uses `this.EndPoint.ToString()` which means `address` and `port` must be the same to end up with the same string).